### PR TITLE
Proper translation from Chinese to English

### PR DESCRIPTION
**Pull request by Timtech4u**
Carefully translated ES6-QuickStart-and-Tips ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -386,16 +389,15 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
-  解析赋值
+   Parsing assignment
 
-  > 可以轻松获取对象、数组等的元素，并赋值到指定变量
+   > You can easily get objects, arrays, and so on, and assign values to the specified variable
 
-  - Array ArrayLike Object 等，具有迭代器接口的对象
-
-  * e.g.
+   - Array Array object, etc., with an iterator interface object
```
Comment (fundon) [inline README.md]: Array, ArrayLike object, etc., with an iterator interface object

**Comment by fundon** [comment]
Rebase the commits, merge them into one commit. Thanks.

**Comment by fundon** [comment]
@Timtech4u Try the command `git rebase -i HEAD~4`.

**Comment by Timtech4u** [comment]
Alright @fundon , I'll do that

**Comment by Timtech4u** [comment]
Done @fundon  , I had to fix some git issues ;-)
screenshot from 2017-09-05 20-57-37


Thanks for allowing me contribute :+1:

**Comment by fundon** [comment]
Thanks

File: README.md [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
 ------------------------
 
-ES6: 快速体验，及实用小贴士
+ES6: quick experience, and practical tips
 
-> 2016 年是 ES6 大力推广和普及的黄金时期，也是今年的流行趋势，
-> 就一个 [ES6](https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓) 关键词，
-> 在 GitHub 上就有这么多搜索结果。（赶紧跟上大部队！）
+> 2016 is ES6 vigorously promote and popularize the golden age, but also this year's fashion trends,
+> On a [ES6] (https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓) keywords,
```
Comment (fundon) [inline README.md:7]: Should `[ES6](https://github.com/search?o=desc&q=ES6&s=stars&type=Repositories&utf8=✓)`, no space

File: README.md [hunk 3]
```diff
@@ -1,62 +1,62 @@
...
+> There are so many search results on GitHub. (Keep up with big troops!)
 
-![It's very hot!](images/github-search-results.png)
+! [It's very hot!] (Images / github-search-results.png)
```
Comment (fundon) [inline README.md:10]: No space!

File: README.md [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
 
 
-## 简介
+## Introduction
 
-ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.ecma-international.org/ecma-262/6.0/index.html)。
+ES6 is short for ECMAScript 6 **, [EC6-262, version 6] (http://www.ecma-international.org/ecma-262/6.0/index.html).
```
Comment (fundon) [inline README.md:15]: No Space!

File: README.md [hunk 5]
```diff
@@ -1,62 +1,62 @@
...
+* What is the relationship between ECMAScript and JavaScript?
 
-  前者是后者的语言标准，后者是前者的一个实现。
+  The former is the latter's language standard, the latter is an implementation of the former.
 
-* ES6 在浏览器，Node.js 支持如何，适不适合开发，生产？
+* ES6 in the browser, Node.js support how suitable for development, production?
 
-  - 浏览器： [ECMAScript 兼容列表](http://kangax.github.io/compat-table/es6/)
+  - Browser: [ECMAScript Compatibility List] (http://kangax.github.io/compat-table/es6/)
```
Comment (fundon) [inline README.md:36]: No Space!

File: README.md [hunk 6]
```diff
@@ -1,62 +1,62 @@
...
 
-  - Node.js：[ES2015 Support](http://node.green)
+  - Node.js: [ES2015 Support] (http://node.green)
```
Comment (fundon) [inline README.md:38]: No Space!

File: README.md [hunk 7]
```diff
@@ -1,62 +1,62 @@
...
 
-  - 性能：[Performance of ES6 features relative to the ES5 baseline operations per second](https://kpdecker.github.io/six-speed/)
+  - Performance: ES6 features relative to the ES5 baseline operations per second] (https://kpdecker.github.io/six-speed/)
```
Comment (fundon) [inline README.md:40]: No Space!

File: README.md [hunk 8]
```diff
@@ -884,49 +882,48 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
-  - 加强对 Unicode 的支持，并且扩展了字符串对象
+   - Enhanced support for Unicode, and extended string objects
 
-  * e.g.
+   * e.g.
 
-    ```js
-    // same as ES5.1
-    "𠮷".length == 2
+     `` `js
```
Comment (fundon) [inline README.md:893]: No Space

File: README.md [hunk 9]
```diff
@@ -884,49 +882,48 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
-    // for-of iterates code points
-    for(var c of "𠮷") {
-      console.log(c);
-    }
-    ```
+     // for-of iterates code points
+     for (var c of "𠮷") {
+       console.log (c);
+     }
+     `` ``
```
Comment (fundon) [inline README.md:910]: No Space, and 3 `

File: README.md [hunk 10]
```diff
@@ -1362,11 +1355,11 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
       ```
 
 
-### [Reflect API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect)
+### [Reflect API] (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect)
```
Comment (fundon) [inline README.md:1358]: No Space!

File: README.md [hunk 11]
```diff
@@ -1440,18 +1432,18 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
 
 ## License
 
-授权：[署名-非商业性使用](https://creativecommons.org/licenses/by-nc/4.0/)
+Authorization: [signed - non-commercial use] (https://creativecommons.org/licenses/by-nc/4.0/)
```
Comment (fundon) [inline README.md:1435]: No Space!

**Pull request merged**